src/main/java/by/onliner/page/components/ShoppingCartPopup.java

```diff
@@ -0,0 +1,43 @@
+package by.onliner.page.components;
+
+import by.onliner.page.AbstractComponent;
+import by.onliner.util.Waiter;
+import org.awaitility.Awaitility;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import static java.util.concurrent.TimeUnit.SECONDS;
+
+public class ShoppingCartPopup extends AbstractComponent {
+
+    @FindBy(className = "product-recommended__sidebar-close")
+    private WebElement shoppingCartPopupCloseButton;
+
+    public ShoppingCartPopup() {
+        super();
+    }
+
+    @Override
+    public void waitForComponentOpened() {
+        Waiter.elementToBeClickable(shoppingCartPopupCloseButton);
+    }
+
+    public boolean visibilityOfShoppingCartPopup() {
```

Comment:
please update method's name, it should contain a verb. Methods which return boolean are supposed to start with 'is'

Reply:
Updated to isShoppingCartPopupVisible

Reply:
Please update its implementation as well

Reply:
Updated